src/Storefront/Controller/RobotsController.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Controller;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Storefront\Page\Robots\RobotsPageLoader;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+/**
+ * @internal
+ */
+#[Route(defaults: ['_routeScope' => ['default']])]
+#[Package('storefront')]
+class RobotsController extends StorefrontController
+{
+    public function __construct(private readonly RobotsPageLoader $robotsPageLoader)
+    {
+    }
+
+    #[Route(path: '/robots.txt', name: 'frontend.robots.txt', defaults: ['_format' => 'txt', '_httpCache' => true], methods: ['GET'])]
+    public function robotsTxt(Request $request): Response
+    {
+        $context = Context::createCLIContext();
```

Comment:
is there a reason, why you do not provide the context as method parameter?

Reply:
When I remember correctly the problem is that we have `_routeScope=default`. There the context is missing.

Reply:
does it need to have the default route scope?

Reply:
okay, I see :+1: 

but it is somehow strange, to use the CLI method here :grin: 

I guess you can use the create default method and put a PHPStan ignore comment above

Reply:
@mstegmeyer perfect that works.

However I am not sure if that follows the principles of how the route scope should be used. But from my side, I am fine with this implementation :-)